docker/test/integration/minifi/processors/PutSplunkHTTP.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, context, schedule={'scheduling strategy': 'EVENT_DRIVEN'}):
             clazz='PutSplunkHTTP',
             properties={
                 'Hostname': 'splunk',
-                'Port': '8088'
+                'Port': '8088',
+                'Token': 'Splunk 176fae97-f59d-4f08-939a-aa6a543f2485'
```

Comment:
Where does this token come from? Can I write "Splunk " followed by any UUID? A comment could be useful to explain.

Reply:
The token is composed of "Splunk" followed by the uuid of the http event collector token which is a uuid. This uuid is the uuid of the default `splunk_hec_token` event collector contained in the splunk docker image.

Reply:
Added comment in 8e49c2e08d1824ec45f6a70a6cf7ea006fa6442b